sensors:
  depth_camera_0:
    type: depth_camera
    pose:
      xyz: [3.082385, 1.750927, 1.82]
      rpy: [0, 'pi/2', 0]

  logical_camera_0:
    type: logical_camera
    pose:
      xyz: [5, 1.750927, 1.82]
      rpy: [0, 'pi/2', 'pi/2']


  laser_profiler_0:
    type: laser_profiler
    pose:
      xyz: [0., 1.486, 1.526]
      rpy: ['pi/2', 'pi/2', 0]

  range_finder_0:
    type: proximity_sensor
    pose:
      xyz: [0, 2.840000, 0.970000]
      rpy: [1.570796, 0.524365, 1.570796]

  breakbeam_0:
    type: break_beam
    pose:
      xyz: [-0.004795, 4.492549, 0.879306]
      rpy: [0, 0, -1.557900]

  logical_camera_12: # over shelf
    type: logical_camera
    pose:
      xyz: [-3.25 - 6*2.04, 0, 2.49]
      rpy: [0, 'pi/2', 'pi/2']
